spec/models/article_spec.rb

```diff
@@ -52,6 +52,39 @@ def build_and_validate_article(*args)
       end
     end
 
+    describe "#validate co_authors" do
+      it "is invalid if the co_author is the same as the author" do
+        article.co_author_ids = [user.id]
+
+        expect(article).not_to be_valid
+      end
+
+      it "is invalid if there are duplicate co_authors for the same article" do
+        co_author1 = create(:user)
+        article.co_author_ids = [co_author1, co_author1]
+
+        expect(article).not_to be_valid
+      end
+
+      it "is invalid if the co_author is entered as a text value rather than an integer" do
+        article.co_author_ids = [user.id, "abc"]
+
+        expect(article).not_to be_valid
+      end
+
+      it "is invalid if the co_author ID is not greater than 0" do
+        article.co_author_ids = [user.id, "0"]
```

Comment:
Should this be `0` instead of `"0"`? Looking at it right now, I'm not sure if this test is not valid because its a string or because of the fact that the id is `0`

Reply:
It should be `0`... that's what I get for copy-pasta-ing. 🤦‍♀️ The test was passing because passing a string is _also_ invalid. Good catch, @vaidehijoshi!